about/index.md

```diff
@@ -19,9 +23,11 @@ Structured Threat Information Expression (STIX™) is a structured language to r
 
 <br class="clear: both;" />
 
-### The STIX Community
+## The STIX Community
+
+STIX is a collaborative effort that is led by DHS but driven by an open community. 
 
-STIX is a collaborative effort that is led by DHS but driven by an open community. Community members make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, specifications, and supporting information. Join the community:
+Join the community to make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, and specifications.
```

Comment:
maybe just "Join the community to make open-source contributions to the STIX language, tools, and documentation"? It reads kind of confusing now (mostly due to confusing language in the original version)